splinepy/io/gismo.py

```diff
@@ -99,27 +96,49 @@ def export(fname, multipatch=None, indent=True):
                 con_spline_id_end,
             )
         ).reshape(-1, 1)
-        # HERE WE SHOULD IDENTIFY THE ORIENTATION
+
+        # Reorder arrays
+        con_spline_id_start = con_spline_id_start[start_order]
+        con_face_id_start = con_face_id_start[start_order]
+        con_spline_id_end = con_spline_id_end[end_order]
+        con_face_id_end = con_face_id_end[end_order]
+
+        # Identify Orientation
+        axis_mapping = np.empty(
+            (start_order.size, multipatch.para_dim), dtype=np.int32
+        )
+        axis_orientation = np.empty(
+            (start_order.size, multipatch.para_dim), dtype=bool
+        )
+        for i, (id_start, start_face, id_end, end_face) in enumerate(
+            zip(
+                con_spline_id_start.flatten(),
+                con_face_id_start.flatten(),
+                con_spline_id_end.flatten(),
+                con_face_id_end.flatten(),
```

Comment:
```suggestion
                con_spline_id_start.ravel(),
                con_face_id_start.ravel(),
                con_spline_id_end.ravel(),
                con_face_id_end.ravel(),
```
`flatten()` copies while `ravel()` tries not to copy.  
gustaf still has a lot of flatten